DigitalLearningSolutions.Web/Views/LearningMenu/Tutorial/Tutorial.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+@model TutorialViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/learningMenu/tutorial.css")" asp-append-version="true">
+
+@{
+  ViewData["HeaderPrefix"] = "";
+  ViewData["Application"] = Model.TutorialContent.CourseTitle;
+  ViewData["Title"] = Model.TutorialContent.CourseTitle;
+  ViewData["CustomisationId"] = @Model.CustomisationId;
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/{Model.CustomisationId}";
+  ViewData["HeaderPathName"] = Model.TutorialContent.CourseTitle;
+}
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">@Model.TutorialContent.Name</h1>
+    <h2 class="nhsuk-u-secondary-text-color nhsuk-u-margin-bottom-0">@Model.TutorialContent.Status @Model.ScoreSummary</h2>
+    <p class="nhsuk-u-secondary-text-color nhsuk-u-margin-bottom-4">@Model.GetTimeSummary</p>
+  </div>
+</div>
+
+@if (Model.CanShowProgress) {
+  <div class="nhsuk-inset-text nhsuk-u-margin-top-0 nhsuk-u-margin-bottom-4">
+    <span class="nhsuk-u-visually-hidden">Information: </span>
+    <p>Based on your diagnostic assessment outcome, this tutorial is <b>@Model.TutorialRecommendation</b>.</p>
+  </div>
+}
+
+@if (Model.TutorialContent.Objectives != null){

```

Comment:
Should have whitespace before opening curly bracket 🙂

Reply:
Whoops! Fixed in 0262ae6 🙂